sdk/attestation/attestation/CHANGELOG.md

```diff
@@ -6,10 +6,11 @@
 
```

Comment:
Should we say something here about the new stuff?

Reply:
What did I miss? I thought I'd captured all the changes in the changelog.

Reply:
I meant something that captures the spirit of the changes listed under breaking changes, e.g. The following methods now have more friendly types. Or something like that but you do not have to force it, I am fine with leaving it blank.

Reply:
Ah, makes more sense.